### PR TITLE
AMkrtchyan_20_01

### PR DESCRIPTION
**Pull request by amiamahy**

### CODE REVIEW

File: homework-20-01-24/Aren-Mkrtchyan/terraform_test/ssh.tf
Comment (Gev544) [inline homework-20-01-24/Aren-Mkrtchyan/terraform_test/ssh.tf:1]: Must be part of ec2.tf

File: homework-20-01-24/Aren-Mkrtchyan/terraform_test/gateway.tf
Comment (Gev544) [inline homework-20-01-24/Aren-Mkrtchyan/terraform_test/gateway.tf:1]: Must be part of Vpc.tf

File: homework-20-01-24/Aren-Mkrtchyan/terraform_test/subnet.tf
Comment (Gev544) [inline homework-20-01-24/Aren-Mkrtchyan/terraform_test/subnet.tf:1]: must be part of vpc.tf

File: homework-20-01-24/Aren-Mkrtchyan/terraform_test/route.tf
Comment (Gev544) [inline homework-20-01-24/Aren-Mkrtchyan/terraform_test/route.tf:1]: must be part of vpc

**Review by Gev544 — CHANGES_REQUESTED** [review]
Use variables for regions, ami etc...
create separate variables.tf file to put vars there.

**Review by amiamahy — COMMENTED** [review]
repaired

**Review by amiamahy — COMMENTED** [review]
added docker files

**Review by shtajiryan — CHANGES_REQUESTED** [review]
Move the docker homework into classwork-13-01-24, make a separate branch and PR for it. The branch logic should be, aren-mkrtchyan-classwork-13-01-24. Revert your latest commit on this branch so I can merge terraform hw.

Also remove files from terraform_test folder, they can be in the root of aren-mkrtchyan inside homework-20-01-24

**Pull request closed without merging**